mlir/lib/Conversion/GPUCommon/GPUToLLVMConversion.cpp

```diff
@@ -605,11 +621,9 @@ void GpuToLLVMConversionPass::runOnOperation() {
                 !module.getTargetsAttr().empty());
       });
 
-  mlir::arith::populateArithToLLVMConversionPatterns(converter, patterns);
-  mlir::cf::populateControlFlowToLLVMConversionPatterns(converter, patterns);
+  // These aren't covered by the ConvertToLLVMPatternInterface right now.
```

Comment:
out of curiosity - why these passes are deleted and not covered?

Reply:
Sorry I didn't follow what you mean by deleted?

The `populateXXXXToLLVMConversionPatterns` that I didn't remove are the one for which we haven't implemented the `ConvertToLLVMPatternInterface`.
The one I removed (`populateControlFlowToLLVMConversionPatterns` and  `populateFuncToLLVMConversionPatterns`) will be handled line 604 above (and only when these dialects are loaded: we only pay what we may need :) )